src/buddy/sign/jws.clj

```diff
@@ -147,5 +147,14 @@
                        {:type :validation :cause :signature})))
      (decode-payload payload))))
 
+(defn decode-no-signature
```

Comment:
Totally agree. Is my namespace placement way off? Felt like maybe it should go in the jwt ns but it needs access to the split into parts function, although that's pretty trivial.